$NetBSD: patch-al,v 1.1 2007/10/13 07:54:31 rillig Exp $

Since C99, the probability that some system header defines the type
"bool" has risen.

--- packages/xpce/src/h/kernel.h.orig	2007-10-13 09:41:23.000000000 +0200
+++ packages/xpce/src/h/kernel.h	2007-10-13 09:42:10.000000000 +0200
@@ -1272,7 +1272,7 @@ NewClass(constant)			/* @nil, @default *
   ABSTRACT_CONSTANT
 End;
 
-NewClass(bool)				/* @on, @off */
+NewClass(swi_bool)			/* @on, @off */
   ABSTRACT_CONSTANT
 End;
 
@@ -1596,8 +1596,8 @@ GLOBAL int	XPCE_mt;		/* we are multi-thr
 GLOBAL struct constant ConstantNil;	/* MUST be first! */
 GLOBAL struct constant ConstantDefault;
 GLOBAL struct constant ConstantClassDefault;
-GLOBAL struct bool     BoolOn;
-GLOBAL struct bool     BoolOff;
+GLOBAL struct swi_bool BoolOn;
+GLOBAL struct swi_bool BoolOff;
 
 GLOBAL Var	RECEIVER;		/* @receiver */
 GLOBAL Var	RECEIVER_CLASS;		/* @receiver_class */
